lib/src/clojure_lsp/refactor/transform.clj

```diff
@@ -774,17 +774,24 @@
              (qualified-symbol? (z/sexpr zloc)))
     (let [z-sexpr (z/sexpr zloc)
           z-name (name z-sexpr)
-          z-ns (namespace z-sexpr)]
-      (if-let [ns-usage (q/find-namespace-usage-by-alias db (shared/uri->filename uri) (symbol z-ns))]
-        (if-let [ns-def (q/find-definition db ns-usage)]
-          (when-not (:external? ns-def)
-            {:ns (:name ns-def)
-             :name z-name})
-          (when-not (:external? ns-usage)
-            {:new-ns (:name ns-usage)
-             :name z-name}))
-        {:new-ns z-ns
-         :name z-name}))))
+          z-ns (namespace z-sexpr)
+          ;; TODO: shouldn't this also look for unaliased ns-usages?
```

Comment:
See #1023

Reply:
Shouldn't we mention the issue on the comment to make it easier to understand the comment?

Reply:
Yeah, sorry... I had meant to do that but I wrote all these comments before writing the issues, and forgot to go back and add the cross references.

Reply:
Cross-references added